azuredeploy.json

```diff
@@ -215,7 +215,7 @@
               "value": "[reference('Root').outputs.adminDesktopVMName.value]"
             },
           "vmSize": {
-            "value": "Standard_D2_v3"
+            "value": "Standard_D2_v2"
```

Comment:
This is necessary but I don't think sufficient. If you do a whole tree search I think you find _v3 in three locations?